components/versionreporting/02-variabes.tf

```diff
@@ -45,7 +37,6 @@ variable "containers_partitions" {
     helmcharts   = "/namespace"
     renovate     = "/repository"
     docsoutdated = "/docTitle"
-    netflow      = "/netflow"
```

Comment:
Assuming this is not needed anymore and not accidental?

Reply:
I believe Felix had it as a poc for something but theres no data